packages/flutter/lib/src/rendering/editable.dart

```diff
@@ -2066,7 +2066,8 @@ class RenderEditable extends RenderBox with RelayoutWhenSystemFontsChangeMixin,
     selectWordsInRange(from: _lastTapDownPosition!, cause: cause);
   }
 
-  /// Selects the set words of a paragraph in a given range of global positions.
+  /// Selects the set words of a paragraph that intersect a given range of global positions.
+  /// The set of words selected are not strictly bounded by the range of global positions.
```

Comment:
The first paragraph should be once sentence per the styleguide.  Maybe just add an empty line of `///` between these two.